src/openrct2/world/Footpath.cpp

```diff
@@ -1929,12 +1936,14 @@ void footpath_update_path_wide_flags(const CoordsXY& footpathPos)
             //}
         }
 
-        if ((pathConnections & FOOTPATH_CONNECTION_NW) && pathList[3] != nullptr && !pathList[3]->AsPath()->IsWide())
+        if ((pathConnections & FOOTPATH_CONNECTION_NW) && std::get<3>(pathList) != nullptr
+            && !std::get<3>(pathList)->AsPath()->IsWide())
         {
             constexpr uint8_t edgeMask1 = EDGE_SE | EDGE_SW;
-            if ((pathConnections & FOOTPATH_CONNECTION_NE) && pathList[7] != nullptr && !pathList[7]->AsPath()->IsWide()
-                && (pathList[7]->AsPath()->GetEdges() & edgeMask1) == edgeMask1 && pathList[0] != nullptr
-                && !pathList[0]->AsPath()->IsWide())
+            if ((pathConnections & FOOTPATH_CONNECTION_NE) && std::get<7>(pathList) != nullptr
```

Comment:
Please put `std::get<7>(pathList)` into a `const auto&` variable.

Reply:
There are a few other examples where a particular path element is accessed 2 or more times, is there a preference/rule on when to extract these usages into a variable?  I don't want to make this more complicated however consistency is important.